packages/core/lib/commands/debug/inlineConfigNetwork.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function (options) {
+  const Config = require("@truffle/config");
+
+  const configObject = Config.default();
+  configObject.networks = {
+    inline_config: {
+      url: options.url,
+      network_id: "*"
+    }
+  };
+  configObject.network = options.network ? options.network : "inline_config";
+  configObject.compileNone = true;
+  configObject._ = [];
```

Comment:
Why is this line here?  Is this necessary?

Reply:
removed